js_modules/dagit/packages/core/src/graph/getFullOpLayout.ts

```diff
@@ -20,7 +19,7 @@ export const getDagrePipelineLayout = memoize(layoutPipeline, _layoutCacheKey);
 
 const _asyncDagrePipelineLayout = (solids: ILayoutOp[], parentSolid?: ILayoutOp) => {
   return new Promise((resolve) => {
-    const worker = new LayoutWorker();
+    const worker = new Worker(new URL('../workers/dagre_layout.worker', import.meta.url));
```

Comment:
I'll trust you on this one - if I remember right the problem here was that this script wasn't preserved as a separate file by webpack without that weird worker-loader thing involved?

Reply:
Webpack 5 no longer requires `worker-loader`, so this should be fine now.